go.mod

```diff
@@ -1,37 +1,37 @@
 module github.com/openshift/cluster-cloud-controller-manager-operator
 
-go 1.22.0
+go 1.22.5
 
-toolchain go1.22.2
+toolchain go1.23.0
```

Comment:
This will break CI I think, can you manually revert just this line?

Reply:
Yes my bad, I'm not sure what I've updated that's done this!